packages/@aws-cdk/core/lib/cfn-resource.ts

```diff
@@ -273,14 +274,24 @@ export class CfnResource extends CfnRefElement {
     this.addPropertyOverride(propertyPath, undefined);
   }
 
+  /**
+   * Indicates that this resource depends on another resource and cannot be
+   * provisioned unless the other resource has been successfully provisioned.
+   *
+   * @deprecated use addDependency
+   */
+  public addDependsOn(target: CfnResource) {
+    return this.addDependency(target);
+  }
+
   /**
    * Indicates that this resource depends on another resource and cannot be
    * provisioned unless the other resource has been successfully provisioned.
    *
    * This can be used for resources across stacks (or nested stack) boundaries
    * and the dependency will automatically be transferred to the relevant scope.
    */
-  public addDependsOn(target: CfnResource) {
+  public addDependency(target: CfnResource) {
```

Comment:
Why are we  adding a new method instead of just changing the body of `addDependsOn`?

Reply:
:confused: I originally did just change the body, but @TheRealAmazonKendra asked me to change the name in a previous review.